Tests/MapKitTests/MKMapViewTests.swift

```diff
@@ -35,7 +35,42 @@ final class MKMapViewTests: XCTestCase {
         let annotationViewWithAnnotation = mapView.dequeueReusableAnnotationView(withClass: MKPinAnnotationView.self, for: annotation)
         XCTAssertNotNil(annotationViewWithAnnotation)
     }
-
+    
+    func testWithEmptyItemArray() {
+        let mapView = MKMapView()
+        
+        let meter = 500.0
+        let emptyItemArray: [CLLocationCoordinate2D] = []
+        let edgePadding: UIEdgeInsets = .init(top: 50, left: 50, bottom: 50, right: 50)
+        
+        let emptyItemInMapView: Void = mapView.zoom(to: emptyItemArray, meter: meter, edgePadding: edgePadding, animated: true)
+        
+        XCTAssertNotNil(emptyItemInMapView)
```

Comment:
This assertion seems not correct, the only thing it is testing is if void is '()' which is always true :))

Reply:
@LucianoPAlmeida What exactly am I supposed to do here? I'm writing the unit test for the first time :(

Reply:
@mustafagunes no problem, I would recommend that after call the method test the state of the map. Like... if you pass x,y,z param which state e.g. region, position, zoom level... the map is expected to be? 
Example:
```swift
mapView.zoom(to: oneItemArray, meter: meter, edgePadding: edgePadding, animated: false)
XCTAssertNotNil(mapView.someState, expected)
```